src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -0,0 +1,2718 @@
+	"mmorpg.word.mustbegear": "Предмет экипировки",
+	"mmorpg.word.noduplicatesockets": "Не повторяет гнёзд",
+	"mmorpg.word.none": "Ничего",
+	"mmorpg.word.normal_gear": "Обычная экипировка",
+	"mmorpg.word.not_affected_by_potent": "Не влияет на потенцию.",
+	"mmorpg.word.nothing": "Ничего",
+	"mmorpg.word.oblivion": "Забвение",
+	"mmorpg.word.per_stack": " (per stack): ",
+	"mmorpg.word.pet_basic": "Обычная атака питомца:",
+	"mmorpg.word.player_affixes": "Аффиксы игрока",
+	"mmorpg.word.potential_cost": "Затраты потенции: %1$s",
+	"mmorpg.word.potion": "Зелье",
+	"mmorpg.word.pressaltforstatinfo": "Нажми Alt для просмотра описания",
+	"mmorpg.word.pressshiftforrequirements": "Нажми Shift для просмотра требований",
+	"mmorpg.word.professions": "Професии",
+	"mmorpg.word.rage": "Ярость",
+	"mmorpg.word.refuge": "Убежище",
+	"mmorpg.word.requirements": "Требования",
+	"mmorpg.word.road": "Дорога",
+	"mmorpg.word.rune": "Руна",
+	"mmorpg.word.salvaging": "Слом",
+	"mmorpg.word.shell": "Оболочка",
+	"mmorpg.word.skillgem": "Навыки",
+	"mmorpg.word.soul": "Душа",
+	"mmorpg.word.spell_stats": "Характеристики заклинания:",
+	"mmorpg.word.spirit": "Дух",
+	"mmorpg.word.splitter": "Разделитель",
+	"mmorpg.word.star": "Звезда",
+	"mmorpg.word.stats": "Характеристики",
+	"mmorpg.word.status_effect": "Статус эффект",
+	"mmorpg.word.suppgem": "Вспомогаюзие камни",
+	"mmorpg.word.tags": "Теги",
+	"mmorpg.word.talents": "Таланты",
+	"mmorpg.word.tier": "Ступень",
+	"mmorpg.word.tower": "Башня",
+	"mmorpg.word.unique_gear": "Уникальная экипировка",
+	"mmorpg.word.unsalvagable": "Неразбераемый",
+	"mmorpg.word.upgradeaffix": "Улучшить аффикс",
+	"mmorpg.word.upgradeenchant": "Улучшить зачарование",
+	"mmorpg.word.upgradequality": "Улучшить качество",
+	"mmorpg.word.upgradesuniquestats": "Добавляет 10% к уникальным характеристикам",
+	"mmorpg.word.usableon": "Подходит для",
+	"mmorpg.word.very_hard": "ОЧень тяжело",
+	"mmorpg.word.ward": "Щит",
+	"mmorpg.word.weapon": "Оружие:",
+	"mmorpg.word.wiki": "Wiki",
+	"mmorpg.word.wind": "Ветер",
+
+"_comment": " [CATEGORY]: WORDS - NAMES",
+
+
+"_comment": " [CATEGORY]: STATS - NAMES",
+
+    "mmorpg.stat.accuracy": "Точность",
+    "mmorpg.stat.ailment_damage": "Урон от болезней",
+    "mmorpg.stat.alchemy_core_drop_bonus": "Бонус ядра",
+    "mmorpg.stat.alchemy_double_drop": "Шанс на удвоенный дроп",
+    "mmorpg.stat.alchemy_exp_bonus": "Бонус опыта профессии",
+    "mmorpg.stat.alchemy_greater_drop_bonus": "Эпический дроп",
+    "mmorpg.stat.alchemy_lesser_drop_bonus": "Обычный дроп",
+    "mmorpg.stat.alchemy_medium_drop_bonus": "Редкий дроп",
+    "mmorpg.stat.alchemy_misc_drop_bonus": "Прочие дропы",
+    "mmorpg.stat.alchemy_triple_drop": "Шанс на тройной дроп",
+    "mmorpg.stat.all_all_damage": "Весь урон",
+    "mmorpg.stat.all_any_wep_damage": "Урон любым оружием",
+    "mmorpg.stat.all_attributes": "Все атрибуты",
+    "mmorpg.stat.all_chaos_damage": "Хаос урон",
```

```suggestion
    "mmorpg.stat.all_chaos_damage": "Урон хаосом",
```
all_fire_damage - Огненный урон
all_chaos_damage - Урон хаосом

---

Судя по твоему комменту я не прошёл тест на робота

Исправлено